wdio-chrome.conf.js

```diff
@@ -189,8 +186,29 @@ exports.config = {
      * @param {Array.<Object>} capabilities list of capabilities details
      * @param {Array.<String>} specs List of spec file paths that are to be run
      */
-    // before: function (capabilities, specs) {
-    // },
+    before: function (capabilities, specs) {
+        browser.addCommand('percyScreenshot', (screenshotName, featureType) => {
+
+            let viewportWidths = [];
+
+            switch(featureType.toLowerCase())
+            {
+                case 'desktop':
+                    viewportWidths = [1280];
+                    break;
+                case 'mobile':
+                    viewportWidths = [414];
+                    break;
+                case 'shared':
+                    viewportWidths = [414, 1280];
```

Comment:
Can we extract these `414`s and `1280`s to `const` explaining what these number mean exactly? I'm quite sure I know, but it might not be that obvious.